testfixtures/logcapture.py

```diff
@@ -106,17 +107,17 @@ def uninstall_all(cls):
         "This will uninstall all existing :class:`LogHandler` objects."
         for i in tuple(cls.instances):
             i.uninstall()
-        
+
```

Comment:
Bytes are cheap, and losing all the whitespace changes would make this pull request much more manageable. Trailing whitespace is a part of pep8 I really coudn't give a monkeys about...